edgelet/iotedge/src/support_bundle.rs

```diff
@@ -176,7 +183,7 @@ where
 
         #[cfg(unix)]
         let inspect = ShellCommand::new("journalctl")
-            .args(&["-u", "iotedge"])
+            .args(&["-u", "-a", "iotedge"])
```

Comment:
This won't work. It needs to be `"-a", "-u", "iotedge"`

Reply:
Hasn't been resolved. (Make sure you don't resolve unless you've actually made the requested change, or added a comment why you're not making the requested change).